atm.js

```diff
@@ -1,3 +1,168 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+  // // console.log("I'm ready")
+//  var checkingBalance = parseFloat($("#checking //.balance").html().split($)[1]);
+//  var checkingInput = parseFloat($("#checking .input").val());
+//  var savingsBalance = parseFloat($("#savings .balance").html().split($)[1]);
+//  var savingsInput = parseFloat($("#savings .input").val());
+
+
+  //var floatInputAmount = parseFloat(inputAmount).toFixed(2);
+  //console.log(typeof(floatInputAmount));
+  // Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+  //Checking deposits
+  //1. Get entry input from users
+  //2. click on deposit, add to total
+  //3.
+
+  //Checking withdraws
+  //3. Get entry from users
+  //4. click on withdraw, subtracts total
+  //5. if total < 0 , reject transaction
+  //6. if total = 0, change color to red
+
+  //Test jQuery is working
+  //Make the <body>'s background color turn red
+//  $("body").css("background", "red");
+
+  //Add a click listener to the checking account's "Deposit" button
+  $(".deposit:first").on("click", function(evt) {
```

Comment:
I think you're looking too hard for the right selector. Always think most simply.